nvflare/fuel/hci/client/fl_admin_api.py

```diff
@@ -396,11 +396,10 @@ def list_jobs(self, options: str = None) -> FLAdminAPIResponse:
             command = command + " " + options
         success, _, reply = self._get_processed_cmd_reply_data(command)
         if success:
-            if reply.get("data"):
-                for data in reply["data"]:
-                    if data["type"] == "table":
-                        details = data["rows"]
-                        return FLAdminAPIResponse(APIStatus.SUCCESS, details, reply)
+            meta = reply.get("meta")
+            if meta:
+                jobs_list = meta.get("jobs", [])
+                return FLAdminAPIResponse(APIStatus.SUCCESS, jobs_list, reply)
```

Comment:
This LGTM, I just went through the code and think we could have some clean-ups regarding fl_admin_api/admin_api/flare_api, but that can be done later.

Reply:
We'll sunset fl_admin_api in future. flare_api is the way to go.